phpstan.neon

```diff
@@ -12,5 +14,9 @@ parameters:
         - '%rootDir%/../../../src/Propel/Generator/Behavior/Delegate/templates/*'
         - '%rootDir%/../../../src/Propel/Generator/Behavior/Sortable/templates/*'
         - '%rootDir%/../../../src/Propel/Generator/Behavior/NestedSet/templates/*'
+        - '%rootDir%/../../../src/Propel/Runtime/Connection/PdoConnection.php'
+        - '%rootDir%/../../../src/Propel/Generator/Command/Helper/ConsoleHelper.php'
+        - '%rootDir%/../../../src/Propel/Generator/Command/InitCommand.php'
     ignoreErrors:
         - '#Call to an undefined method .+Collection::.+Array\(\)#'
+        - '#Class .+TreeBuilder constructor invoked with 0 parameters, 1-3 required#'
```

Comment:
This one will be with us for a long time. Is it possible to inline some of these ignores into the file itself?

Reply:
I dont think so. This is the best we can do for now. And doesnt hurt too much here.